src/main/java/uk/gov/digital/ho/hocs/aws/SQSConfiguration.java

```diff
@@ -41,8 +41,8 @@ public AmazonSQS sqsClient(@Value("${case.creator.sqs.access-key}") String acces
     }
 
     @Bean
-    public SQSQueuePrefix queuePrefix(@Value("${case.creator.sqs.region}") String region,
-                                      @Value("${case.creator.sqs.account-id}") String accountId) {
+    public SQSQueuePrefix queuePrefix(@Value("${aws.sqs.region}") String region,
+                                      @Value("${aws.account.id}") String accountId) {
```

Comment:
AWS_ACCOUNT_ID is not set in kube-hocs-case-creator. Would suggest adding it in deploy.sh as follows:

if [[ ${KUBE_NAMESPACE} == *prod ]]
then
    export AWS_ACCOUNT_ID="658718335966"
    export MIN_REPLICAS="2"
    export MAX_REPLICAS="6"
else
    export AWS_ACCOUNT_ID="449472082214"
    export MIN_REPLICAS="1"
    export MAX_REPLICAS="3"
fi